account_bank_statement_import/models/account_bank_statement_import.py

```diff
@@ -132,7 +160,7 @@ def _import_statement(self, stmt_vals):
         return self._create_bank_statement(stmt_vals)
 
     @api.model
-    def _parse_file(self, data_file):
+    def _parse_file(self, dummy_data_file):
```

Comment:
@NL66278 please rename and add `# pylint: disable=W0613` (or W0612?) to silence pylint about the parameter